CONTRIBUTING.md

```diff
@@ -0,0 +1,67 @@
+# Contributing to `wonder-stuff`
+
+🙇Thank you for your interest in contributing to the Wonder Stuff repository. However, **we are not currently accepting community contributions** on this project.
+
+The participation of others is a wonderful 🎁gift. When we are ready to accept that gift, we will update these guidelines.
+If you have already been invited to participate, the remainder of these guidelines are for you.
+
+📖 Be sure to read our [Code of Conduct](https://github.com/Khan/render-gateway/blob/master/CODE_OF_CONDUCT.md).
+
+## 🛑 Bugs And Feature Requests
+
+⚠️ **We are not currently accepting externally raised bugs and feature requests**
+
+## 💻 Code Changes
+
+⚠️ **We are not currently accepting externally provided code changes**
```

Comment:
Now that we're using GitHub internally for code reviews.  How do we differentiate code reviews from KA employees vs. external people that try to create code reviews?  I wonder if there's maybe some way that we can leverage Gerald to verify that a PR is coming from a KA employee and auto-close ones that don't.  No action req'd.